### ISSUE TITLE
no-consecutive-blank-lines warning for the first line

### ISSUE THREAD

**Issue by verekia** [wooorm/remark-lint#115]
Hello!

The `no-consecutive-blank-lines` rule warns when the first line of a file is empty:

```md
<!-- Invalid -->

# Main Title

Hello
```

I think the first line should not be treated as "consecutive lines" and have its case handled separately. Leaving an empty first line or not comes down to personal preference of course. I personally think a markdown file is much more appealing with its main title not squeezed at the top of the editor. A line of padding does wonder for the eyes.

I can offer help if needed.
